app/subscribe/views.py

```diff
@@ -34,6 +35,23 @@ def subscribe():
                     user_guid=user_guid,
                     new_value={"email_body": email_body}
                 ))
+                # Email for the user
+                if form.user_email.data:
+                    email_to_user = [form.user_email.data]
+                    unsubscribe_link = url_for('unsubscribe.unsubscribe', _external=True)
+                    email_user_body = render_template('emails/new_subscriber_user.html',
+                                                      first_name=form.user_first.data,
+                                                      last_name=form.user_last.data,
+                                                      unsubscribe_link = unsubscribe_link)
+                    send_email(subject="Confirmation Email",
+                               sender=current_app.config['MAIL_SENDER'],
+                               recipients=email_to_user,
```

Comment:
You can just use `[form.user_email.data]` here and remove line 40 since the value is used once.